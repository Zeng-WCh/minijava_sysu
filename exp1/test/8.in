0-0-8+3+9+1-3-9+4-9-3+0-5+7-8-7-9-8-5+5+7+6+4+2-8-3+1+5+1+2+6-3+1+9-4-9-1+5+0-9-8-8+1+7+7-5-4+4-2-5+8-5-8-6-8-2-9-5-1+8-5+3-4+3-8-6-3-5+8-6+5-8+2+4-7+0-9+0+4-6-1-0-4+3+0-1+0+9+7+0-5-9+8+1-7-0+4-5+9+3