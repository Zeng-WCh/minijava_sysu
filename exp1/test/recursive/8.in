8-5-0-2-1-4+1+4-8-7+2-1-7-3-5-6+1-8-3-3+8-4+0-6+8-8+3+3-9+0