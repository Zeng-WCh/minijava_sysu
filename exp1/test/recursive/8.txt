5-7-3-8+3-3-6+3+2-7-1-3-1-7-3-3+4-0+0-4-5-1+1+2+0-9-5+8-4+7