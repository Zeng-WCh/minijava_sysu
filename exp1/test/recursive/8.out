Input an infix expression and output its postfix notation:
85-0-2-1-4-1+4+8-7-2+1-7-3-5-6-1+8-3-3-8+4-0+6-8+8-3+3+9-0+
End of program.
